src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -784,6 +778,142 @@ public void orderByAscFieldWithSpaceTest() throws IOException {
         Assert.assertEquals("The list is not in ascending order", sortedTestFields, testFields);
     }
 
+    @Test
+    public void testWhereWithBoolEqualsTrue() throws IOException {
+        JSONObject response = executeQuery(
+                    StringUtils.format("SELECT * " +
+                                    "FROM %s/account " +
+                                    "WHERE male = true ORDER BY age LIMIT 5",
+                            TestsConstants.TEST_INDEX_BANK)
+        );
+
+        JSONArray hits = getHits(response);
+        Assert.assertThat(hits.length(), equalTo(4));
+    }
+
+    @Test
+    public void testWhereWithBoolEqualsTrueAndGroupBy() throws IOException {
+        JSONObject response = executeQuery(
+                String.format(Locale.ROOT, "SELECT * " +
```

Comment:
You've changed to StringUtils in previous tests, but forgot to do that in the rest :)

I recommend to get rid of formatting at all and inline everything, also consider changing format of the query to 

SELECT 
FROM
WHERE
GROUP BY
LIMIT

Reply:
I should inline everything on the integration tests as well?